.github/workflows/gradle.yml

```diff
@@ -237,6 +238,37 @@ jobs:
       - name: Ensure no file change
         run: git --no-pager diff && test -z "$(git --no-pager diff)"
 
+      - name: Publish Connectors Base Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        id: connectors-test-results
+        if: always()
+        with:
+          junit_files: "/actions-runner/_work/airbyte/airbyte/*/build/test-results/*/*.xml\n/actions-runner/_work/airbyte/airbyte/*/*/build/test-results/*/*.xml"
+          comment_mode: off
+          json_file: connectors_base_results.json
+          json_test_case_results: true
+          check_name: "Connectors Base Test Results"
+          
+      - name: Setup Cloud SDK
```

Comment:
nit: I don't know how this step is named elsewhere, I found `Cloud SDK` a bit confusing with `airbyte-cloud`, is it worth renaming to `Google Cloud SDK`?

Reply:
👍